<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CV Absence Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BTN_Manage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_OpenSession">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Session</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQUit">
    <property name="title">
     <string>Quit</string>
    </property>
   </widget>
   <addaction name="menuQUit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
